protobufs/account/v2/device.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package account.v2;
+
+message Device {
+  string model = 1;
+  string serial = 2;
+  repeated uint32 linked_pids = 3;
```

Comment:
PIDs can, technically, be larger than this so I think moving forward we should increase this to a uint64. Other areas of these protobufs still use the smaller uint32 size though and we can't change that without breaking backwards compatibility. So maybe it would be better to only up the size to a uint64 in a potential future v3?

@DaniElectra opinions?

Reply:
That sounds reasonable to me 👍 

Alright disregard this then